lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced-tabbed.component.spec.ts

```diff
@@ -0,0 +1,215 @@
+import { ContentTestingModule } from '../../../testing/content.testing.module';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { SearchDateRangeAdvanced } from './search-date-range-advanced/search-date-range-advanced';
+import { SearchFilterTabbedComponent } from '../search-filter-tabbed/search-filter-tabbed.component';
+import { SearchDateRangeAdvancedComponent } from './search-date-range-advanced/search-date-range-advanced.component';
+import { SearchDateRangeAdvancedTabbedComponent } from './search-date-range-advanced-tabbed.component';
+import { DateRangeType } from "./search-date-range-advanced/date-range-type";
+import { InLastDateType } from "./search-date-range-advanced/in-last-date-type";
+import { endOfDay, formatISO, parse, startOfDay } from "date-fns";
+
+@Component({
+    selector: 'adf-search-filter-tabbed',
+    template: ``
+})
+export class MockSearchFilterTabbedComponent {}
+
+@Component({
+    selector: 'adf-search-date-range-advanced',
+    template: ``
+})
+export class MockSearchDateRangeAdvancedComponent {
+    @Input()
+    dateFormat: string;
+    @Input()
+    maxDate: string;
+    @Input()
+    field: string;
+    @Input()
+    initialValue: SearchDateRangeAdvanced;
+
+    @Output()
+    changed = new EventEmitter<Partial<SearchDateRangeAdvanced>>();
+    @Output()
+    valid = new EventEmitter<boolean>();
+}
+describe('SearchDateRangeAdvancedTabbedComponent', () => {
+    let component: SearchDateRangeAdvancedTabbedComponent;
+    let fixture: ComponentFixture<SearchDateRangeAdvancedTabbedComponent>;
+    let betweenMockData: SearchDateRangeAdvanced, inLastMockData: SearchDateRangeAdvanced, anyMockDate: SearchDateRangeAdvanced;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            declarations: [SearchDateRangeAdvancedTabbedComponent, SearchFilterTabbedComponent, SearchDateRangeAdvancedComponent],
+            imports: [
+                TranslateModule.forRoot(),
+                ContentTestingModule
+            ],
+            providers: [
+                { provide: SearchFilterTabbedComponent, useClass: MockSearchFilterTabbedComponent },
+                { provide: SearchDateRangeAdvancedComponent, useClass: MockSearchDateRangeAdvancedComponent }
+            ]
+        });
+        fixture = TestBed.createComponent(SearchDateRangeAdvancedTabbedComponent);
+
+        component = fixture.componentInstance;
+        component.id = 'dateRangeAdvanced';
+        component.context = {
+            queryFragments: {
+                dateRangeAdvanced: ''
+            },
+            update: jasmine.createSpy('update')
+        } as any;
+        component.settings = {
+            hideDefaultAction: false,
+            dateFormat: 'dd-MMM-yy',
+            maxDate: 'today',
+            field: 'createdDate, modifiedDate',
+            displayedLabelsByField: {
+                'createdDate': 'Created Date',
+                'modifiedDate': 'Modified Date'
```

Comment:
no need for quotes at the left side

Reply:
These has been resolved in [this](https://github.com/Alfresco/alfresco-ng2-components/pull/8714/commits/bdaa0e52fd3a31ba7c3802c6cdc6a27a733d6057) commit